app/services/waste_carriers_engine/registration_completion_service.rb

```diff
@@ -8,11 +8,13 @@ def run(transient_registration)
       @transient_registration = transient_registration
 
       transient_registration.with_lock do
+        prepare_finance_details_for_lower_tier
+
         copy_names_to_contact_address
         copy_data_from_transient_registration
 
         set_reg_identifier
-        set_expiry_date
+        set_expiry_date if registration.upper_tier?
```

Comment:
🤔 Why are we setting the expiry date here, and again in the CanChangeRegistrationService when we activate?

Reply:
yes :) We have been doing that before and that's the same thing that happens on the old service. I spoke to Alan about it few weeks ago and we decided to keep this behaviour. Basically, we always have an expiry date, but we update it when the registration is activated. Same as old service :)

Reply:
Ahh, that makes sense! Thanks